src/app/storage/types/storage.ts

```diff
@@ -0,0 +1,19 @@
+import { AxiosError, AxiosResponse } from 'axios';
+
+import { AllStoredQueriesObject } from '../query/types';
+
+export type OverallStorageObject = AllStoredQueriesObject; // & AllStored<TYPEs>Object etc.
```

Comment:
this corrects the nested type redundancy mentioned before, in an extensible manner